core/scss/components/lockup/_lockup.scss

```diff
@@ -0,0 +1,147 @@
+@charset "UTF-8";
+
+//
+// Lockup
+//
+// A lockup with the Stanford wordmark logo and department/unit name. It is used in the header of a website and links to the homepage of the department/unit. When building a department lockup, besides the Stanford wordmark, the most obvious element should be the most granular relevant department.
+//
```

Comment:
I would love to display the different variants too! The problem is the conditional in the twig file. Currently I have it check if line 2 is present, if line 2 is being defined in the json file, it automatically selects the two-line variant, and if not, it will use the default one-line. I asked @sherakama if there is a way to pass multiple json objects, but he said KSS doesn't offer that as an option unless we set up a custom "example" section. 

When I first started it, I didn't use the conditional in the twig file so I have both variants displayed, but it seems the conditional is nice to have. Let me know your thoughts on this! Perhaps @JBCSU might have some thoughts too. I guess it depends on how or where we want to handle the logic part (not very familiar with that myself)? 

I applied your other change by the way. Thanks for catching that!

Reply:
Ah, right. That makes total sense.

I agree that the conditional is nice to have in the Twig file and outweighs (in this case) the immediate need to show the variants in the pattern library. It's also better to show all of the options populated rather than just line one, so passing through the examples in the JSON file is also probably the right thing to do here.

I saw that the other change has been applied. That said, there is nothing blocking this from being merged IMO.

Reply:
Cool - and thanks for the review! I'm pushing up a commit for a small fixup in the comments. After that, please feel free to merge whenever you like. 

And I forgot to say this yesterday, but please feel free to add line 3/4 or make changes as needed while I'm on vacation. I don't own this component 😄

Reply:
See my below comment as well.